tasks/main.yml

```diff
@@ -22,3 +22,20 @@
 - name: Create /etc/passwd.client
   template: src=passwd.client dest=/etc/exim4/passwd.client group=Debian-exim
   notify: update exim4.conf
+
+- name: Add headers to outgoing emails
+  template: src="conf.d/{{ item }}" dest=/etc/exim4/conf.d/{{ item }}
+  with_items:
+    - acl/30_exim4-config_check_rcpt
+    - acl/31_exim4-config_check_not_smtp
+    - main/03_exim4-config_extra_options
+  when: (exim4_outgoing_headers is defined) and exim4_outgoing_headers
```

Comment:
If the files do more than adding headers, will the result of not copying them create the exact same Exim4 behaviours except for the added headers? If not, then the task should always rules, and the template themselves should take care of not adding header (they already do that).